docs/reference/glossary.asciidoc

```diff
@@ -54,19 +54,26 @@
   pairs. The value can be a simple (scalar) value (eg a string, integer,
   date), or a nested structure like an array or an object. A field is
   similar to a column in a table in a relational database.
-  +
   The <<glossary-mapping,mapping>> for each field has a field _type_ (not to
   be confused with document <<glossary-type,type>>) which indicates the type
   of data that can be stored in that field, eg `integer`, `string`,
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is a non-scoring <<glossary-query,query>>, meaning that it does not score documents.
+  It is only concerned about answering the question - "Does this document match?". 
+  The answer is always a simple, binary yes or no. This kind of query is said to be made 
+  in a https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html["filtering" context], 
```

Comment:
I think you can use a relative link like <<query-filter-context,filter context>> here.

Reply:
I tried it is not working

Reply:
I also just tried, works for me. What are you doing to check the resulting page? Have you tried building the whole reference doc, e.g. by using something like `build_docs.pl --doc docs/reference/index.asciidoc --out ~/temp/asciidoc/  --open` where build_docs.pl points to my local checkout of https://github.com/elastic/docs?